src/VisualStudio/Core/Def/AnalyzerDependency/AnalyzerDependencyCheckingService.cs

```diff
@@ -169,9 +171,11 @@ public void ReanalyzeSolutionForConflicts()
                     }
                 }
 
-                hostDiagnosticUpdateSource.UpdateDiagnosticsForProject(project.Id, s_dependencyConflictErrorId, builder.ToImmutable());
+                hostDiagnosticUpdateSource.UpdateAndAddDiagnosticsArgsForProject(ref argsBuilder.AsRef(), project.Id, s_dependencyConflictErrorId, builder.ToImmutable());
             }
 
+            hostDiagnosticUpdateSource.RaiseDiagnosticsUpdated(argsBuilder.ToImmutableAndClear());
```

Comment:
>argsBuilder [](http://example.com/codeflow?start=63&length=11)

check size?

Reply:
➡️ `RaiseDiagnosticsUpdated` checks the count itself